src/NUnitFramework/tests/Constraints/CollectionSupersetConstraintTests.cs

```diff
@@ -69,6 +69,71 @@ public void HonorsIgnoreCase(IEnumerable expected, IEnumerable actual)
             }
         }
 
+#if !NET35
```

Comment:
No more NET35 in this release, so not needed.

Reply:
Oops. Thanks for catching that!